platform/platform-app/src/main/resources/features.xml

```diff
@@ -902,6 +902,28 @@
             mvn:ddf.platform.solr/platform-solr-server-standalone/${project.version}/txt/synonyms
         </configfile>
 
+        <configfile finalname="/data/solr/attributes/conf/solr.xml">
```

Comment:
These should really go in the catalog app but currently platform has the majority of the solr config files listed.